airflow/kubernetes/kube_client.py

```diff
@@ -107,23 +109,34 @@ def get_kube_client(
     if conf.getboolean("kubernetes_executor", "enable_tcp_keepalive"):
         _enable_tcp_keepalive()
 
+    new_client_config = _get_default_configuration()
```

Comment:
Nit: you can make the configuration smaller by keeping this variable 
```suggestion
    configuration= _get_default_configuration()
```

Reply:
Thank you. I still see the same number of changes, however pushing the patch for this